Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue3049.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST && __IOS__
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 3049, "DisplayActionSheet freezes app in iOS custom renderer (iPad only)", PlatformAffected.iOS)]
+	public class Issue3049 : TestContentPage
```

Comment:
What does this iOS test have to do with Android API changes?

Reply:
might need rebase' seem it was changed the base branch